tests/test_backup_restore.py

```diff
@@ -34,8 +35,11 @@ def test_reading_the_instruction_contents(
 ) -> None:
     """Test reading the content of the .HA_RESTORE file."""
     with (
-        mock.patch("pathlib.Path.read_text", return_value=content),
-        mock.patch("pathlib.Path.exists", return_value=exists),
+        mock.patch(
+            "pathlib.Path.read_text",
+            return_value=content,
+            side_effect=FileNotFoundError if not exists else None,
```

Comment:
I'd change the parameters instead to be the return value and side effect or just the side effect, to avoid encoding logic to the parameter.